pokemongo_bot/event_handlers/logging_handler.py

```diff
@@ -4,7 +4,10 @@
 
 
 class LoggingHandler(EventHandler):
-    def handle_event(self, event, sender, level, data):
+    def handle_event(self, event, sender, level, formatted_msg, data):
```

Comment:
What do we need this `formatted_msg` for? It seems like it would make logging more complicated if things are sending event, formatted_msg and data.